r/NAMESPACE

```diff
@@ -308,6 +311,8 @@ importFrom(rlang,seq2)
 importFrom(rlang,set_names)
 importFrom(rlang,syms)
 importFrom(rlang,warn)
+importFrom(stats,median)
+importFrom(stats,quantile)
```

Comment:
Does this require that we add `stats` to `Imports:` in DESCRIPTION?

Reply:
Good catch, I think yes.

Reply:
Added in d697d7c9f947dcf856430d7b78accfbe81149790